src/components/FullscreenGrid/index.tsx

```diff
@@ -2,8 +2,18 @@ import * as React from 'react';
 import makeClassName from 'classnames';
 
 import { AnyReactNode } from '../../typeUtils';
+import ContentShellComponent from './ContentShell';
 import styles from './styles.module.scss';
 
+type PublicProps = {
+  children?: AnyReactNode;
+  className?: string;
+};
+
+type Props = PublicProps;
+
+export type PanelAttribs = 'altSidePanel' | 'mainSidePanel';
```

Comment:
I know this is old code, but could this be an `enum` instead?

Reply:
Yes!